adapters/Vungle/VungleAdapter/GADMAdapterVungleDelegate.h

```diff
@@ -27,6 +27,7 @@ typedef NS_ENUM(NSUInteger, GADMAdapterVungleAdType) {
 typedef NS_ENUM(NSUInteger, BannerRouterDelegateState) {
   BannerRouterDelegateStateRequesting,
   BannerRouterDelegateStateCached,
+  BannerRouterDelegateStateWillPlaying,
```

Comment:
Add a new state `BannerRouterDelegateStateWillPlaying` which represents the state that the Banner view is successfully returned by calling `addAdViewToView:withOptions:placementID:placementID:error:` method.

Reply:
Can this be updated to `BannerRouterDelegateStateWillPlay`? Thanks

Reply:
@crbinsf Got it, thanks a lot for the instructions. 🙂